src/routes/blog/post/case-study-majik-kids/+page.markdoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Majik Kids: Building an audio-based content platform for children"
+description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, illustrators, etc., to build an alternative content platform with a focus on audio storytelling, creating a healthier learning experience for children.
+date: 2024-01-19
+cover: /images/blog/majik-kids.png
+timeToRead: 5
+author: aditya-oberai
+category: case-studies
+---
+
+# Ideating an alternative content platform for children
+
+Being an entrepreneur in the entertainment-based education space for over 15 years and a father of a child of age 7, Bradley T. Morris from Salt Spring Island, Canada, recognized the harmful effects that excessive screen time and screen addiction can have on the nervous system of his child. He and his wife had discovered already that engaging their son, Sauryn, with audio stories while allowing them to participate in activities such as drawing, painting, etc., did not cause similar withdrawal symptoms that screen addiction causes. 
```

Comment:
```suggestion
Being an entrepreneur in the entertainment-based education space for over 15 years and a father of a child of age 7, Bradley T. Morris from Salt Spring Island, Canada, recognized the harmful effects that excessive screen time and screen addiction can have on the nervous system of his child. He and his wife had discovered already that engaging their son, Sauryn, with audio stories while allowing them to participate in activities such as drawing or painting did not cause similar withdrawal symptoms that screen addiction causes. 
```

Same here. "Such as", "including", "as well as" are all phrases that mean "etc.", so we can skip it here.

Reply:
Ah good point.